dpipe/im/grid.py

```diff
@@ -50,6 +50,27 @@ def get_boxes(shape: AxesLike, box_size: AxesLike, stride: AxesLike, axis: AxesL
         yield make_box_([start, np.minimum(start + box_size, shape)])
 
 
+def make_batch(divide_iterator: Iterable, batch_size: int = 1):
```

Comment:
Do u really need defining this function? Wouldn't https://more-itertools.readthedocs.io/en/stable/api.html#more_itertools.batched be enough with casting to numpy?

Reply:
Done in new commit, now use batched